rectools/models/sasrec.py

```diff
@@ -1131,17 +1215,17 @@ def __init__(  # pylint: disable=too-many-arguments, too-many-locals
         use_key_padding_mask: bool = False,
         dropout_rate: float = 0.2,
         session_max_len: int = 32,
-        loss: str = "softmax",
+        loss: Union[Literal["softmax", "BCE", "gBCE"], LossCalculatorBase] = "softmax",
```

Comment:
Let's put `Literal["softmax", "BCE", "gBCE"]` into a separate var at the top

Reply:
fixed